libs/portals/my-pages/health/src/screens/MedicinePrescriptions/components/PrescriptionsTable.tsx

```
@@ -0,0 +1,247 @@
+        items={
+          prescriptions?.map((item, i) => ({
+            id: `${item.id}-${i}`,
+            medicine: item?.name ?? '',
+            usedFor: item?.indication ?? '',
+            process: item?.amountRemaining ?? '',
+            validTo: formatDate(item?.expiryDate) ?? '',
+            status: undefined,
+            // ------------------------------------- //
+            // TODO: Add this back when the API is ready
+            // ------------------------------------- //
+            lastNode: true //item?.isRenewable
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)